### ISSUE TITLE
Issue with Dark mode no longer automatically making new websites dark

### ISSUE THREAD

**Issue by hookedmermaid** [darkreader/darkreader#6427]
As the title says.

Yesterday I updated chrome and since then, I've been having issues with websites not being in dark mode. Many of these are websites I visit daily, so the change is strange. 

Normally, when I visit a new website for the first time, as I have dark mode always on, the website shows in dark mode. Now, unless I add the website to the list, it shows as light mode. I've never had to do that before, and honestly, until yesterday I had no idea what the top left toggle/button actually did as everything was always in dark mode already. 

I'm working in low lighting/at night and I have visual processing issues, so having to be blinded every time I want to see a new website is, as you can imagine, a bit of a problem. I have no idea what changed with the update, but I've tried near everything I can think on my side to get it to work how it did before and... nothing.

![drissue](https://user-images.githubusercontent.com/19909953/128353821-706129ad-2527-423e-871d-3956e88029b2.PNG)

**Comment by Gusted** [comment]
Hi @hookedmermaid!

Did you by any chance enabled the `Invert listed only` under the Site list tab?

Regards,
Gusted

**Comment by hookedmermaid** [comment]
Hi Gusted

I have had a few things in there over the last year, but when it started acting up deleted everything listed in there to see if it would change anything. It didn't.

There's also now an error showing up on the extension. 

![drerror](https://user-images.githubusercontent.com/19909953/128495804-5605b980-e239-4ff1-a2ff-677e4afec634.PNG)

Kind regards

Nora

**Comment by Gusted** [comment]
> I have had a few things in there over the last year, but when it started acting up deleted everything listed in there to see if it would change anything. It didn't.

You most likely have "Invert listed only" enabled, which means dark reader only enables on the sites that you specify.
You have to select the "Not invert listed" option.
![image](https://user-images.githubusercontent.com/25481501/128502703-98963b1d-a2b3-4d27-aefc-ec7a0b5bf67e.png)

**Comment by hookedmermaid** [comment]
No. I changed nothing with the extension, and it's definitely set to Invert.

I updated chrome (because it was being crap) and when I reloaded it all my websites were showing in light mode. I never view websites in light mode due to vision problems. I checked the settings and the only way I found to get websites to show in dark mode (with the extension turned on) is to add them to the Invert only list. 

Then I noticed it was showing the above error when I went into chromes extensions manager.

**Comment by hookedmermaid** [comment]
Just removed the extension and readded it. 

Seems to be working as expected for now. I'm not sure what the error was or why it stopped working, but it seems okay now.

**Comment by bershanskiy** [comment]
@hookedmermaid This might be far-fetched, but what is the name of extension to the left of Dark Reader? 
![image](https://user-images.githubusercontent.com/45960703/128521586-603ae671-4f1c-403a-b4a1-543c13dc5a43.png)

**Comment by hookedmermaid** [comment]
Adblocker. I’ve had it forever and it’s never caused a problem with Dark
Reader.
